testapi.pm

```diff
@@ -173,7 +173,8 @@ Saves screenshot of current SUT screen.
 
 =cut
 
-sub save_screenshot () { $autotest::current_test->take_screenshot }
+sub is_serial_terminal;
```

Comment:
Just a minor point, I would have put this to the other stub definitions at the top.

Reply:
I'll fix it

Reply:
https://github.com/os-autoinst/os-autoinst/pull/2287